File: backend/src/main/java/com/unina/ratatouille23/ordinazione/entity/OrdinazioneDTO.java
package com.unina.ratatouille23.ordinazione.entity;

public class OrdinazioneDTO {
    private Ordinazione ordinazione;
    private String operazione;

    public OrdinazioneDTO() {
    }

    public OrdinazioneDTO(Ordinazione ordinazione, String operazione) {
        this.ordinazione = ordinazione;
        this.operazione = operazione;
    }

    public Ordinazione getOrdinazione() {
        return this.ordinazione;
    }

    public void setOrdinazione(Ordinazione ordinazione) {
        this.ordinazione = ordinazione;
    }

    public String getOperazione() {
        return this.operazione;
    }

    public void setOperazione(String operazione) {
        this.operazione = operazione;
    }



}

File: client/lib/models/Ordinazione_DTO.dart
import 'package:ratatouille23/models/Ordinazione.dart';

class Ordinazione_DTO{
  Ordinazione _ordinazione = Ordinazione();
  String _operazione = "";


  Ordinazione_DTO(this._ordinazione, this._operazione);

  Ordinazione_DTO.fromJsonSTOMP(Map<String, dynamic> json){
    _operazione = json['operazione'];
    _ordinazione = Ordinazione.fromJson(json['ordinazione']);
  }

  Ordinazione get ordinazione => _ordinazione;

  set ordinazione(Ordinazione value) {
    _ordinazione = value;
  }

  String get operazione => _operazione;

  set operazione(String value) {
    _operazione = value;
  }
}